ehitchhiking-ui/src/app/pages/registration/registration.component.sass

```diff
@@ -0,0 +1,79 @@
+@import "../../../styles"
+
+.user-info
+  background: $mat-badge-font-weight
+  border-radius: 0 0 2% 2%
+  padding: 5%
+  position: relative
+  border: none
+  margin: 0
+  box-shadow:  0 0 0.5rem 0.1rem rgba(221, 221, 221, 1)
+
+.hover-info
+  color: mat-color($primary, 900)
+  text-decoration: none
+.forg-pass:hover
+ color: mat-color($primary, 100)
+
+.center
+  display: flex
+  justify-content: center
+
+#login-form
+  margin: auto
+  width: 40%
+  padding: 5%
+
+.page
+  display: flex
+  width: 100%
+  height: 100%
+  background: mat-color($mat-blue-gray)
+  align-items: center
+
+div .exadel-login
+  background-color: mat-color($accent, 50)
+  border-radius: 2% 2% 0 0
+  padding: 5%
+  img
+    width: 50%
+    margin-left: 25%
+
+.form-img
+  i
+    font-size: $subheading-1
+    margin-right: 0.2rem
+
+.input
+  font-size: $input
+  left: 20%
+  border: 1px solid #dcdcdc
```

Comment:
use one of the existing color constant or create a new one

Reply:
fix